Source/WebCore/platform/graphics/GraphicsTypesGL.h

```diff
@@ -91,6 +92,15 @@ struct GCGLSpan {
     GCGLSpan(const std::array<T, Extent>& array)
         : data(array.data())
     { }
+    GCGLSpan(const GCGLSpan& other) noexcept
+        : GCGLSpan(other.data)
+    { }
+    GCGLSpan& operator=(const GCGLSpan& other) noexcept
```

Comment:
The implicit assignment operator should be used.
Memcopying over const fields is never ok.
(you cannot have const fields for types that support assignment)

Reply:
OK